HelpSource/Classes/Server.schelp

```diff
@@ -342,6 +342,10 @@ subsection:: Message Bundling
 
 Server provides support for automatically bundling messages. This is quite convenient in object style, and ensures synchronous execution. See also link::Guides/Bundled-Messages::
 
+method:: makeWindow
+Instantiates the "old school" SuperCollider server window, that was used to show up by default on startup up to version 3.5.
+in case you are running supercollider on emacs, it makes a server view made of emacs widgets. Also, in Mac OS X, this method will most likely work out over makeGui
```

Comment:
Grammar/tone check – "Instantiates the "old-school" SuperCollider server window that used to appear by default on startup prior to version 3.5. In case you are running SuperCollider on Emacs, this function makes a server view from Emacs widgets. Also, in macOS, this ..."